Makefile

```diff
@@ -8,16 +8,16 @@ docker: version
 
 .PHONY: devshell  # Open a developer shell in the docker env
 devshell: docker
-	docker run --rm -it -v $$PWD:/opt --entrypoint /bin/bash $(IMAGE)
```

Comment:
I don't think removing "-it" will change anything, but wanted someone with more docker experience to confirm.

Reply:
Yeah we need this. The use case is that developers type `make devshell` at the command line and end up in an interactive shell with all the dependencies to run tests. Was `-it` giving you trouble?

Reply:
Oh I see. Maybe it was causing trouble in the `test-*` targets below. Can you try with just `-t`? I like that b/c then it preserves color in the logs, which makes the test output a bit more legible. But if it doesn't work with -t just leave `-it` for the devshell and eliminate it for the test- targets.

Reply:
Yep exactly, it was messing up the testing part but I should have noticed that wasn't a test. Updated for only the test-* and it works correctly.